packages/webapp/src/components/Finances/Transaction/Mobile/styles.module.scss

```diff
@@ -18,8 +18,14 @@
   margin-bottom: 85px;
 }
 
+.transactionDate {
+  font-size: 12px;
+  line-height: 16px;
+  padding-top: 16px;
+}
```

Comment:
Very small, but looks like there is a missing 8px gap between the date and a selected transaction:

<img width="485" alt="Screenshot 2023-10-31 at 3 04 59 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/e40ce27d-e480-49f8-88c8-dc50cc045547">

(I'm basing the gap off of [this Figma Screen](https://www.figma.com/file/96NZ02oYe3jpet1roUp0s0/Mockups?type=design&node-id=5385-27574&mode=design&t=Kryx8zvD328u0JQ0-0))

Since the spacing between date and item is already good on non-selected transactions, maybe the `exandableItem` padding has to be decreased some corresponding amount? Hopefully it's possible without messing with the list items too much...!

Reply:
Thank you so much!!! I forgot to check it with the expanded view.
I think it looks good with the padding of `exandableItem` `10px`, but it goes against the 8px rule, right...?
I added the 8px gap, and I'll ask Loïc for suggestions.

Reply:
I think you could 🤭... break the rule if you like 😈

Reply:
I really want to 😈😈
But I am also curious what Loïc will say :)